### DISCUSSION TITLE
My Server is Down, Can Anyone Please Help?

### DISCUSSION THREAD

**Discussion by darrylmorley** [caprover/caprover#1653] [Q&A]
I was trying to force add my domain for a redirect and now all of my sites have gone down can anybody please help?

**Comment by githubsaturn** [comment]
Did you make nginx changes? What do you mean by force add

**Reply by darrylmorley** [reply]
@githubsaturn  Sorry,

I was trying to get help everywhere.

This is related to https://github.com/caprover/nginx-redirect/issues/4

**Comment by yiiman-dev** [comment]
if anyone has problem with down server and dont up nginx or leaved swarm node, i can help as mini project to return that
